bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboAccountHandler.java

```diff
@@ -0,0 +1,333 @@
+import org.openhab.binding.sensibo.internal.dto.setacstateproperty.SetAcStatePropertyRequest;
+import org.openhab.binding.sensibo.internal.dto.settimer.SetTimerReponse;
+import org.openhab.binding.sensibo.internal.dto.settimer.SetTimerRequest;
+import org.openhab.binding.sensibo.internal.model.AcState;
+import org.openhab.binding.sensibo.internal.model.SensiboModel;
+import org.openhab.binding.sensibo.internal.model.SensiboSky;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.TypeAdapter;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * The {@link SensiboAccountHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Arne Seime - Initial contribution
+ */
+@NonNullByDefault
+public class SensiboAccountHandler extends BaseBridgeHandler {
+    private static final int MIN_TIME_BETWEEEN_MODEL_UPDATES_MS = 30_000;
+    private static final int SECONDS_IN_MINUTE = 60;
+    public static String API_ENDPOINT = "https://home.sensibo.com/api";
+    private final Logger logger = LoggerFactory.getLogger(SensiboAccountHandler.class);
+    private final HttpClient httpClient;
+    private final JsonParser jsonParser = new JsonParser();
+    private final RequestLogger requestLogger;
+    private final Gson gson;
+    private SensiboModel model = new SensiboModel(0);
+    private @Nullable ScheduledFuture<?> statusFuture;
+    private @NonNullByDefault({}) SensiboAccountConfiguration config;
+
+    public SensiboAccountHandler(final Bridge bridge, final HttpClient httpClient) {
+        super(bridge);
+        this.httpClient = httpClient;
+
+        gson = new GsonBuilder().registerTypeAdapter(ZonedDateTime.class, new TypeAdapter<ZonedDateTime>() {
+            @Override
+            public void write(final @NonNullByDefault({}) JsonWriter out, final ZonedDateTime value)
+                    throws IOException {
+                out.value(value.toString());
+            }
+
+            @Override
+            public ZonedDateTime read(final @NonNullByDefault({}) JsonReader in) throws IOException {
+                return ZonedDateTime.parse(in.nextString());
+            }
+        }).setLenient().setPrettyPrinting().create();
+
+        requestLogger = new RequestLogger(bridge.getUID().getId(), gson);
+    }
+
+    private boolean allowModelUpdate() {
+        final long diffMsSinceLastUpdate = System.currentTimeMillis() - model.getLastUpdated();
+        return diffMsSinceLastUpdate > MIN_TIME_BETWEEEN_MODEL_UPDATES_MS;
+    }
+
+    public SensiboModel getModel() {
+        return model;
+    }
+
+    @Override
+    public void handleCommand(final ChannelUID channelUID, final Command command) {
+        // Ignore commands as none are supported
```

Comment:
If this thing has any channels then you still need to support the RefreshCommand

Reply:
No channels